koku/api/cloud_accounts/cloud_accounts.json

```
@@ -0,0 +1,14 @@
+[
+    {
+        "name": "AWS",
+        "value": "589173575009",
+        "description": "Cost Management's AWS account ID",
+        "updated_timestamp": "2020-03-06T14:41:11.808752Z"
+    },
+    {
+        "name": "AWS_LOCAL",
+        "value": "589173575009",
+        "description": "Cost Management's AWS_LOCAL account ID",
+        "updated_timestamp": "2020-03-06T14:41:11.808752Z"
+    }
```

AWS_LOCAL should not be in this file.
```suggestion
    }
```

This should match what we previously returned from the API:
```
GET /api/cost-management/v1/cloud-accounts/
HTTP 200 OK
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept

{
    "meta": {
        "count": 1
    },
    "links": {
        "first": "/api/cost-management/v1/cloud-accounts/?limit=10&offset=0",
        "next": null,
        "previous": null,
        "last": "/api/cost-management/v1/cloud-accounts/?limit=10&offset=0"
    },
    "data": [
        {
            "name": "AWS",
            "value": "589173575009",
            "description": "Cost Management's AWS account ID",
            "updated_timestamp": "2019-11-12T15:58:44.886349Z"
        }
    ]
}
```

---

added